VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen.tsx

```diff
@@ -107,14 +107,14 @@ const NotificationsSettingsScreen: FC<NotificationsSettingsScreenProps> = ({ nav
   return (
     <FeatureLandingTemplate backLabel={t('settings.title')} backLabelOnPress={navigation.goBack} title={t('notifications.title')}>
       <Box mb={contentMarginBottom}>
-        {!systemNotificationsOn && alert()}
+        {!systemNotificationsOn && <Box mb={standardMarginBetween}>{alert()}</Box>}
```

Comment:
I adjusted the margin because I noticed the `Personalize Notifications` title was a little too close to the alert box